src/DIRAC/DataManagementSystem/Agent/FTS3Agent.py

```diff
@@ -98,14 +93,18 @@ def __readConf(self):
         # lifetime of the proxy we download to delegate to FTS
         self.proxyLifetime = self.am_getOption("ProxyLifetime", PROXY_LIFETIME)
 
+        # Find out if send to Accounting and/or Monitoring
+        self.monitoringOptions = (
+            opHelper().getValue("DataManagement/MonitoringBackends", "Accounting").replace(" ", "").split(",")
```

Comment:
```suggestion
            opHelper().getValue("DataManagement/MonitoringBackends", ["Accounting"])
```
If I understand correctly the value will go through this: https://github.com/DIRACGrid/DIRAC/blob/fadfa2933c0ffa9a19d701661742de8e4ad67b88/src/DIRAC/ConfigurationSystem/private/ConfigurationClient.py#L163